_posts/en/newsletters/2024-01-03-newsletter.md

```diff
@@ -0,0 +1,402 @@
+    enters the mempool when its parent is unconfirmed.  Correspondingly,
+    a parent transaction may have a low feerate by themselves, but when
+    their descendants' fees are also considered, the feerate might be
+    significantly higher, leading them to confirm faster than expected.
+    Bitcoin Core's fee estimations don't compensate for this situation.
+
+    Cluster mempool will keep related transactions together and support
+    dividing them into chunks that will be profitable to mine together.
+    Ismail suggests tracking the feerates of chunks rather than
+    individual transactions (though a chunk can be a single transaction)
+    and then attempting to find those same chunks in blocks.  If a chunk
+    is confirmed, then fee estimations are updated using its chunk
+    feerate rather than the feerates of individual transactions.
+
+    The proposal was well received, with developers discussing the
+    details an updated algorithm would need to consider.
+
+- **How to specify unspendable keys in descriptors:** Salvatore Ingala
+  started a [discussion][ingala undesc] on Delving Bitcoin about how to allow
+  [descriptors][topic descriptors], particularly those for [taproot][topic
+  taproot], to specify a key for which no private key is known
+  (preventing spending from that key).  One important context for this
+  is sending money to a taproot output that can only be spent via a
+  scriptpath spend.  To do this, the key that allows keypath spending
+  must be set to a unspendable key.
+
+    Ingala describes several challenges to using unspendable keys in
+    descriptors and several proposed solutions with different tradeoffs.
+    Pieter Wuille summarizes several recent in-person discussions about
+    descriptors, including a [particular][wuille undesc2] idea about
+    unspendable keys.  Josie Baker asks for details about why the
+    unspendable key can't be a constant value (such as the
+    nothing-up-my-sleeve (NUMS) point in BIP341), which would allow
+    everyone to immediately know that an unspendable key was used---a
+    possible advantage to some protocols, such as [silent
+    payments][topic silent payments].  Ingala replies to Baker that "it
+    is a form of fingerprinting. You can always reveal this information
+    yourself if you want/need it, but it’s great if the standards don’t
+    force you to do so."  Wuille further replies with an algorithm for
+    generating the proof.  In the last post in the thread at the time of
+    writing, Ingala notes that some of the work of specifying policies
+    related to unspendable keys can be split between descriptors and
+    [BIP388][] wallet policies.
+
+- **V3 transaction pinning costs:** Peter Todd [posted][todd v3] to the
+  Bitcoin-Dev mailing list an analysis of the proposed [v3 transaction
+  relay][topic v3 transaction relay] policy on [transaction pinning][topic
+  transaction pinning] for contract protocols such as LN.  For example,
```

Comment:
Keeping as-is, but feel free to edit before merging.  The example is chosen so that it applies both now and with v3 tx relay.  A few sentences later, when we discuss Bitcoin Core's current rules (plus package relay), we say "Using Bitcoin Core's current default...".  Then, in the next paragraph when we looking at the effect of v3 rules, we introduce that by saying "In the v3 ...".